modules/base/rendering/pointcloud/renderablepointcloud.cpp

```diff
@@ -1511,9 +1511,7 @@ void RenderablePointCloud::updateBufferData() {
 }
 
 void RenderablePointCloud::updateSpriteTexture() {
-    bool shouldUpdate = _hasSpriteTexture && _spriteTextureIsDirty;
-
-    if (!shouldUpdate) {
+    if (!(_hasSpriteTexture && _spriteTextureIsDirty)) [[likely]] {
```

Comment:
I personally find this a lot harder to read. Maybe get rid of the parenthesis if you don't want the variable?

```suggestion
    if (!_hasSpriteTexture || !_spriteTextureIsDirty) [[likely]] {
```
but all the negating makes it harder to read than with the variable IMO